doc/ignition.md

```diff
@@ -37,7 +37,7 @@ following section.
 
 ### Specification ###
 
-- **version** (integer): the version number of the spec. Must be `1`.
+- **ignitionVersion** (integer): the version number of the spec. Must be `1`.
```

Comment:
Somewhere you need to mention this is the only required field and the rest are optional